amundsen_application/static/js/components/common/BadgeList/index.spec.tsx

```diff
@@ -4,7 +4,7 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
 
-import Flag from 'components/common/Flag';
+import { ClickableBadge } from 'components/common/Badges';
```

Comment:
To get the tests to pass update to `import ClickableBadge from 'components/common/Badges';`

Reply:
is it running unit test just by `npm run test` locally which has the same effect of running it on CI?

Reply:
@feng-tao yes.